<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCityDialog</class>
 <widget class="QDialog" name="EditCityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Overlays</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>City</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbCity"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Latitude:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editLatitude"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Longitude:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editLongitude"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Available Overlays</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="toolTip">
      <string>Select any overlays to be used with this city.</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Selected Overlay</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MyTextEdit" name="edDescription"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Opacity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbOpacity">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>65</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Min Zoom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbMinZoom"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>MaxZoom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbMaxZoom">
       <property name="maximum">
        <number>18</number>
       </property>
       <property name="value">
        <number>18</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTextEdit</class>
   <extends>QTextEdit</extends>
   <header>mytextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditCityDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditCityDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
